railties/lib/rails/paths.rb

```diff
@@ -175,6 +175,12 @@ def to_ary
         @paths
       end
 
+      def expanded_paths
```

Comment:
I can't see it though, we'll probably need new methods in the class. Probably like this:

```ruby
def extensions
  $1.split(',') if @glob =~ /\{([\S]+)\}/
end

def expand_paths_without_uniq # :nodoc:
  raise "You need to set a path root" unless @root.path
  
  map { |path| File.expand_path(path, @root.path) }
end

# Expands all paths against the root and return all unique values.
def expanded
  result = []

  expand_paths_without_uniq.each do |p|
    if @glob && File.directory?(path)
      Dir.chdir(path) do
        result.concat(Dir.glob(@glob).map { |file| File.join path, file }.sort)
      end
    else
      result << path
    end
  end

  result.uniq!
  result
end
```